### PR TITLE
Bring internal use directives into correspondence with style doc

### PR DESCRIPTION
**Pull request by mulkieran**
Do any alphabetizing necessary.

Signed-off-by: mulhern <amulhern@redhat.com>

### CODE REVIEW

**Comment by mulkieran** [comment]
@agrover @trgill review?

File: src/deviceinfo.rs [hunk 1]
```diff
@@ -4,12 +4,11 @@
...
-
-use consts::{DM_NAME_LEN, DmFlags, DM_UUID_LEN};
-use device::Device;
-use dm::{DmName, DmUuid};
-use util::slice_to_null;
+use super::consts::{DM_NAME_LEN, DmFlags, DM_UUID_LEN};
+use super::device::Device;
+use super::dm::{DmName, DmUuid};
+use super::dm_ioctl as dmi;
+use super::util::slice_to_null;
```
Comment (trgill) [inline src/deviceinfo.rs]: why add super:: in these cases?

**Comment by mulkieran** [comment]
It's the rule we decided to use for stratisd. But, I think I might as well deal w/ consts question as well.

**Comment by mulkieran** [comment]
I'll leave dealing w/ consts to separate PR, since they're distinct.

**Pull request merged**